File: examples/v2/default-organization/python/__main__.py
import pulumi
import pulumi_pulumiservice.v2 as ps_v2

config = pulumi.Config()
organization_name = config.get("organizationName") or "service-provider-test-org"

default = ps_v2.DefaultOrganization(
    "default",
    org_name=organization_name,
)

pulumi.export("defaultOrg", organization_name)
pulumi.export("defaultOrgGitHubLogin", default.git_hub_login)

File: examples/v2/default-organization/go/main.go
package main

import (
	v2 "github.com/pulumi/pulumi-pulumiservice/sdk/go/pulumiservice/v2"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
	"github.com/pulumi/pulumi/sdk/v3/go/pulumi/config"
)

func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		cfg := config.New(ctx, "")
		serviceOrg := cfg.Get("serviceOrg")
		if serviceOrg == "" {
			serviceOrg = "service-provider-test-org"
		}

		def, err := v2.NewDefaultOrganization(ctx, "default", &v2.DefaultOrganizationArgs{
			OrgName: pulumi.String(serviceOrg),
		})
		if err != nil {
			return err
		}

		ctx.Export("defaultOrg", def.OrgName)
		return nil
	})
}
